src/govuk/components/exit-this-page/exit-this-page.mjs

```diff
@@ -1,21 +1,50 @@
 /* eslint-disable es-x/no-function-prototype-bind -- Polyfill imported */
 
-import { nodeListForEach } from '../../common.mjs'
+import { normaliseDataset } from '../../common/normalise-dataset.mjs'
+import { nodeListForEach, mergeConfigs, extractConfigByNamespace } from '../../common.mjs'
+import { I18n } from '../../i18n.mjs'
 import '../../vendor/polyfills/Element/prototype/classList.mjs'
 import '../../vendor/polyfills/Element/prototype/dataset.mjs'
 import '../../vendor/polyfills/Function/prototype/bind.mjs'
 
+/**
+ * @constant
+ * @type {ExitThisPageTranslations}
+ * @see Default value for {@link ExitThisPageConfig.i18n}
+ * @default
+ */
+var EXIT_THIS_PAGE_TRANSLATIONS = {
+  keypressProgress: 'Exit this Page key press %{press} of 3',
+  keypressComplete: 'Exit this page activated',
+  helpText: 'press <kbd>Shift</kbd> 3 times<span class="govuk-visually-hidden"> to exit the page</span>'
```

Comment:
Mega nitpick time: VO's text display is displaying the announcement from "3 times" like so:

>...3 timesto exit this page

It's interpreting it in the audio announcement fine but moving the space out of the `span` or putting a `&nbsp;` in there should sort this out:

```suggestion
  helpText: 'press <kbd>Shift</kbd> 3 times <span class="govuk-visually-hidden">to exit the page</span>'
```

Reply:
Have done so 👍